components/tokens/custom-spectrum/custom-large-vars.css

```diff
@@ -18,4 +18,7 @@ governing permissions and limitations under the License.
   --spectrum-edge-to-visual-only-100: 9px;
   --spectrum-edge-to-visual-only-200: 13px;
   --spectrum-edge-to-visual-only-300: 16px;
+
```

Comment:
Double check the tokens updates in here - it seems unrelated to 11ty.

Reply:
Yes this is not related to 11ty. I double checked from the main branch. There is a new line from this file itself.
The below is the same file from the main branch. 
<img width="1144" alt="Screenshot 2023-01-30 at 11 25 33 AM" src="https://user-images.githubusercontent.com/8790510/215398265-15e9c1a8-f583-435e-96ee-d68e11674b59.png">